index.html

```diff
@@ -1009,20 +1016,36 @@
             <var>b</var> as a <a>USVString</a>. The <a>user agent</a> MUST use a serialization
             method that does not branch based on the value of <var>b</var>.
             </li>
-            <li>Add an entry to <var>keys</var> whose key name is the name of <var>i</var> and
-            whose value is <var>s</var>.
+            <li>Add an entry to the <a data-link-for="PushSubscriptionJSON">keys</a> property of
+            <var>json</var> whose key is the name of <var>i</var> and whose value is <var>s</var>.
             </li>
           </ol>
         </li>
-        <li>Add an entry to <var>map</var> whose key name is <code>keys</code> and whose value is
-        <var>keys</var>.
-        </li>
-        <li>Return <var>map</var>.
-        </li>
+        <li>Return <var>json</var>.
       </ol>
+
+      <p>
+        A <dfn>PushSubscriptionJSON</dfn> dictionary represents the <a>JSON type</a> of a
+        <a>PushSubscription</a> that can be converted into a JSON string.
+      </p>
+      <p>
+        The <dfn data-dfn-for="PushSubscriptionJSON">endpoint</dfn> contains the serialized value of
```

Comment:
I'm uneasy about using the serialized value term, as it's not defined anywhere. I'd use the "underlying value" here and again reference https://heycam.github.io/webidl/#get-the-underlying-value.